metadata.yml

```diff
@@ -0,0 +1,241 @@
+      annotations:
+        container.seccomp.security.alpha.kubernetes.io/nginx: localhost/test
+      labels:
+        app: nginx-seccomp
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+    ----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-allowed5
+      annotations:
+        container.seccomp.security.alpha.kubernetes.io/nginx: docker/default
+      labels:
+        app: nginx-seccomp
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+    ```
+
+    While the following pods will be rejected:
+
+    ```yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-disallowed
+      labels:
+        app: nginx-seccomp
+    spec:
+      securityContext:
+        seccompProfile:
+          type: Unconfined
+      containers:
+      - name: nginx
+        image: nginx
+      - name: nginx2
+        image: nginx
+    ----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-disallowed2
+      annotations:
+        container.seccomp.security.alpha.kubernetes.io/nginx: localhost/test2
+      labels:
+        app: nginx-seccomp
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+      - name: nginx2
+        image: nginx
+    ----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-disallowed3
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: unconfined
+      labels:
+        app: nginx-seccomp
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+    ----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-disallowed4
+      annotations:
+        container.seccomp.security.alpha.kubernetes.io/nginx: unconfined
+      labels:
+        app: nginx-seccomp
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+      - name: nginx2
+        image: nginx
+    ----
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx-seccomp-disallowed5
+      labels:
+        app: nginx-seccomp
+    spec:
+      securityContext:
+        seccompProfile:
+          type: Localhost
+          localhostProfile: test2
+      containers:
+      - name: nginx
+        image: nginx
+      - name: nginx2
+        image: nginx
+    ```
+
+    ## License
+
+    ```
+    Copyright (C) 2021 José Guilherme Vanz <jvanz@jvanz.com>
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    ```
```

Comment:
I would drop that, it's useless info and the license is already covered by a dedicated metadata field above

Reply:
You dropped this text from the `README`, but not from the metadata :)

Reply:
:facepalm: